pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -445,11 +445,18 @@ func (a *ofFlowAction) Note(notes string) FlowBuilder {
 	return a.builder
 }
 
-func (a *ofFlowAction) SendToController(reason uint8) FlowBuilder {
+// SendToController will send the packet to the OVS controller.
+// If pause option is true, the packet will be not only sent to the controller, but
+// also paused in the pipeline. The controller could use a resume message to resume
+// this packet letting it continue its journey in the pipeline from where it was
+// paused.
+func (a *ofFlowAction) SendToController(userdata []byte, pause bool) FlowBuilder {
```

Comment:
Better to explain what's the expected content of userdata.

Reply:
Done.